src/Tribe/Cost_Utils.php

```diff
@@ -63,6 +63,13 @@ public function get_event_costs( $event ) {
 
 		$costs = tribe_get_event_meta( $event->ID, '_EventCost', false );
 
+		$costs = array_filter(
+			(array) $costs,
+			static function ( $cost ) {
+				return $cost !== '';
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 6caa12b5d522122a435582ab13004845</sub>